iosApp/iosApp/Pages/TripDetails/TripDetailsStopView.swift

```diff
@@ -26,9 +26,28 @@ struct TripDetailsStopView: View {
                         Spacer()
                         UpcomingTripView(prediction: upcomingTripViewState, routeType: routeType)
                     }
+                    .accessibilityElement(children: .combine)
+                    .accessibilityAddTraits(.isHeader)
+                    .accessibilityHeading(.h2)
                 }
             )
             scrollRoutes
+                .accessibilityLabel(scrollRoutesAccessibilityLabel)
+        }
+    }
+
+    var scrollRoutesAccessibilityLabel: String {
+        if stop.routes.isEmpty {
+            return ""
+        } else if stop.routes.count == 1 {
+            return "Connection to \(stop.routes.first!.shortName) \(stop.routes.first!.type.typeText(isOnly: true))"
+        } else {
+            let lastConnection = stop.routes.last
+            return stop.routes.prefix(stop.routes.count - 1).map {
+                "\($0.shortName) \($0.type.typeText(isOnly: true))"
```

Comment:
suggestion: use `$0.label` here as well, and include "Connections to"

Reply:
Looks like the only piece left is including the "Connections to " prefix when there are plural connections